README.md

```diff
@@ -8,7 +8,22 @@ are published.
 
 If you'd like to stay informed about synthetic monitoring development please visit [our synthetic monitoring page](https://www.elastic.co/what-is/synthetic-monitoring) where you can sign up to be notified of our initial release.
 
+### To Run Example Jouney inline
+
+`npm i`
+
+`npm run build`
+
+`npm link`
+
+`cat examples/inline/short.js | npx @elastic/synthetics --inline`
+
+### To Run Example Journey with Docker and hearbeat, 
+Data will be pushed to ES, which can be viewed using uptime app
+
+`node ./examples/docker/run.sh 7.10.0  -E output.elasticsearch.hosts=["http://localhost:9200"] -E output.elasticsearch.username=admin -E output.elasticsearch.password=changeme`
```

Comment:
Bettet to move this under `examples/docker/readme.md` and also remove `node` from there

Reply:
i think since it's pretty small for now. keep it at root makes sense

Reply:
++ that `node` breaks this. I would prefer we send people to the docs, mainly because READMEs tend not to be updated very frequently, and the docs are mostly done.

That said, I think it'd be a great idea to improve the README for the specific example. If the README is in the same folder it's much more likely to be updated.

Reply:
i already removed node thingy, if @vigneshshanmugam is doing a separate readme after package publgish, we can close this, he mentioned that.